vscode_extension/src/extension.ts

```diff
@@ -1,16 +1,33 @@
-import { Disposable, ExtensionContext, tasks } from "vscode";
+import { ExtensionContext, window, commands } from "vscode";
 import { AacLanguageServerClient } from "./AacLanguageServer";
-import { AacTaskProvider } from "./AacTaskProvider";
+import { executeAacCommand, getAaCVersion } from "./aacExecutableWrapper";
+import { getOutputChannel } from "./outputChannel";
 
-let aacTaskProvider: Disposable | undefined;
 let aacLspClient: AacLanguageServerClient = AacLanguageServerClient.getLspClient();
 
+const EXECUTE_AAC_COMMAND_NAME = "aac.execute";
+
 export function activate(context: ExtensionContext) {
-    aacTaskProvider = tasks.registerTaskProvider(AacTaskProvider.aacType, new AacTaskProvider());
-    aacLspClient.startLanguageServer(context);
+
+    getAaCVersion().then(installedAaCVersion => {
+        if (installedAaCVersion) {
+            aacLspClient.startLanguageServer(context);
+            context.subscriptions.push(
+                commands.registerCommand(EXECUTE_AAC_COMMAND_NAME, executeAacCommand)
+            );
+        } else {
+            const missingAacMessage = `Please install AaC locally to activate
+                these plugin features.\n 'pip install aac'`;
+
+            commands.registerCommand(EXECUTE_AAC_COMMAND_NAME, () => {
+                window.showErrorMessage(missingAacMessage);
+            });
```

Comment:
This can be simplified

```suggestion
            commands.registerCommand(EXECUTE_AAC_COMMAND_NAME, () => window.showErrorMessage(missingAacMessage));
```